graph.js

```diff
@@ -180,7 +180,15 @@ app
       await keytar.setPassword('graphprotocol-auth', node, accessToken)
       logger.status('Access token set for Graph node:', node)
     } catch (e) {
-      logger.error('Failed to set access token:', e)
+      if (process.platform === 'win32') {
+        logger.error(`Error storing Graph access token in Windows Credential Vault:`, e)
+      } else if (process.platform === 'darwin') {
+        logger.error(`Error storing Graph access token in macOS Keychain:`, e)
+      } else if (process.platform === 'linux') {
+        logger.error(`Error storing Graph access token with libsecret (usually gnome-keyring or ksecretservice):`, e)
```

Comment:
This line is a little too long. You can break the string up with `+` or by using backslashes to spread it across multiple lines.